main.go

```diff
@@ -20,15 +27,23 @@ func main() {
 		log.Fatal("Usage: tldr <command name>\n")
 	}
 
+	var page string
+	var err error
+
+	cache := newCache()
 	command := os.Args[1]
-	page, err := fetchPage(getPlatform(), command)
 
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(-1)
+	if page, err = cache.read(command); err != nil {
+		if page, err = fetchPage(getPlatform(), command); err != nil {
+			fmt.Fprintf(os.Stderr, "Command not found: %s\n", command)
```

Comment:
unlike previous version, you assume an error is a not found.
Also any cache error is treated as a cache miss and not printed out, even when the cache has richer errors (see comments on cache errors)